docs/modules/ROOT/pages/vshn-managed/forgejo/index.adoc

```diff
@@ -0,0 +1,38 @@
+= VSHNForgejo (Forgejo)
+:toc: left
+:toclevels: 2
+
+[WARNING]
+====
+VSHNForgejo is currently in alpha. This means that the software is still under active development and may undergo significant changes. Users should be aware that there may be bugs, incomplete features, and potential instability. We appreciate your feedback and encourage you to report any issues you encounter.
```

Comment:
```suggestion
VSHNForgejo is currently in alpha. This means that the service is still under active development and may undergo significant changes. Users should be aware that there may be bugs, incomplete features, and potential instability. We appreciate your feedback and encourage you to report any issues you encounter.
```

You could also use an Antora tag and include the same snippet in multiple pages: https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/